src/Symfony/Component/Intl/Tests/CountriesTest.php

```diff
@@ -535,6 +536,8 @@ public function testGetCountryCodes()
      */
     public function testGetNames($displayLocale)
     {
+        IntlTestHelper::requireFullIntl($this);
```

Comment:
For the `en` locale, we don't require the full intl (and it would be great to support testing this against the polyfill to ensure it stays the case)

Reply:
good idea 👍